feature-libs/organization/administration/components/shared/item.service.ts

```diff
@@ -109,4 +111,10 @@ export abstract class ItemService<T> {
   getRouterParam(key: string): Observable<string> {
     return this.currentItemService.getRouterParam(key);
   }
+  /**
+   * Sets to true when the user is on the entity item form page
+   */
+  setEditMode(isInEdit: boolean) {
```

Comment:
Shoudln't we cover this with unit tests?

Reply:
yes indeed